syntax = "proto3";

package sbai_system_alert_protos;

import "sbai_system_alert_protos/system_alert.proto";

message SystemReport {
    repeated sbai_system_alert_protos.SystemAlert alerts = 1;
    string robot_id = 2;
}
